api-extractor/src/definitions/ApiItem.ts

```diff
@@ -291,31 +280,34 @@ abstract class ApiItem {
    * the documentation.
    */
   protected onResolveReferences(): void {
+    let originalJsDoc: string = '';
+    if (this.jsdocNode) {
+      originalJsDoc = TypeScriptHelpers.getJsDocComments(this.jsdocNode, this.reportError);
+    }
+
     this.documentation = new ApiDocumentation(
-      this.docComment,
+      originalJsDoc,
       this.extractor.docItemLoader,
       this.extractor,
       this.reportError
     );
     // TODO: this.collectTypeReferences(this);
 
-    if (this.documentation.apiTagCount > 1) {
-      this.reportError('More than one API Tag was specified');
-    }
+    const summaryTextCondensed: string = DocElementParser.getAsText(
+      this.documentation.summary,
+      this.reportError).replace('  ', ' ');
```

Comment:
This only replaces the first occurrence.  We have something like Utilities.replaceAll()